src/cleanvision/utils/utils.py

```diff
@@ -12,6 +12,9 @@
 except ImportError:  # pragma: no cover
     PSUTIL_EXISTS = False
 
+# max number of processes that can be forked/spawned for multiprocessing
+MAX_PROCS = 5000
```

Comment:
move this to `utils/constants.py`

Reply:
which dataset did you test it on?

Reply:
yup moved to `utils/constants.py`.

I didn't actually observe the slowdown for this repo, because I didn't have any dataset with millions of images to test on. I made this change because when doing multiprocessing for `find_label_issues` in the cleanlab repo, we observed that when the number of processes gets into the millions suddenly the code becomes slow. So this is more a preventative change, I don't think it hurts.

Reply:
Okay cool